src/app/pages/fund/components/receive-funds-item.tsx

```diff
@@ -0,0 +1,53 @@
+import BitcoinIcon from '@assets/images/btc-icon.png';
+import QRCodeIcon from '@assets/images/fund/qr-code-icon.png';
+import ReceiveFundsEllipses from '@assets/images/fund/receive-funds-ellipses.png';
+import StacksIcon from '@assets/images/fund/stacks-icon.png';
+import { FundPageSelectors } from '@tests/selectors/fund.selectors';
+import { Box } from 'leather-styles/jsx';
+
+import { CurrencySymbol } from '../fund';
+import { FundAccountTile } from './fund-account-tile';
+
+const cryptoDescriptions = {
+  stx: {
+    title: 'Receive STX from a friend or deposit from a separate wallet',
+    iconComponent: (
+      <>
+        <Box>
+          <img src={StacksIcon} width="40px" />
+        </Box>
+        <Box>
+          <img src={ReceiveFundsEllipses} width="24px" />
+        </Box>
+      </>
+    ),
+  },
+  btc: {
+    title: 'Receive BTC from a friend or deposit from a separate wallet',
+    iconComponent: (
+      <>
+        <Box>
+          <img src={BitcoinIcon} width="40px" />
+        </Box>
+        <Box>
+          <img src={ReceiveFundsEllipses} width="24px" />
+        </Box>
+      </>
+    ),
+  },
+};
```

Comment:
Maybe better to define these as components passed in as props, rather than config containing JSX?